util/types/overflow.go

```diff
@@ -18,6 +18,15 @@ import (
 
 	"fmt"
 	"github.com/juju/errors"
+	"github.com/pingcap/tidb/terror"
+)
+
+const (
+	CodeUintAdditionOperationOverflow terror.ErrCode = 1
```

Comment:
@shenli  this pr is not ready yet. I am still doing more work. It is my bad not saying this in this PR. As you can see, the todo list is still marked as todo.